lib/plugin.rb

```diff
@@ -6,6 +6,11 @@ module CocoaPodsKeys
   class << self
     include FileUtils
 
+    # Register for the pre-install hooks to setup & run Keys
+    Pod::HooksManager.register('cocoapods-keys', :pre_install) do
+      CocoaPodsKeys.setup
```

Comment:
this should now call `setup` with the options, instead of pulling off of `Pod::Config.instance.podfile`